types/test.tsx

```diff
@@ -1,3 +1,6 @@
+/** @jsx x */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {Attributes, Element} from 'xast'
```

Comment:
I don't think these need to be imported, dtslint does some magic to auto detect/import types used in expectations.